src/rng/backends/rocrand/mrg32k3a.cpp

```diff
@@ -0,0 +1,867 @@
+ * without specific prior written permission.
+ *
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ * You are under no obligation whatsoever to provide any bug fixes, patches,
+ * or upgrades to the features, functionality or performance of the source
+ * code ("Enhancements") to anyone; however, if you choose to make your
+ * Enhancements available either publicly, or directly to Lawrence Berkeley
+ * National Laboratory, without imposing a separate written license agreement
+ * for such Enhancements, then you hereby grant the following license: a
+ * non-exclusive, royalty-free perpetual license to install, use, modify,
+ * prepare derivative works, incorporate into other computer software,
+ * distribute, and sublicense such enhancements or derivative works thereof,
+ * in binary and source code form.
+ *
+ * If you have questions about your rights to use or distribute this software,
+ * please contact Berkeley Lab's Intellectual Property Office at
+ * IPO@lbl.gov.
+ *
+ * NOTICE.  This Software was developed under funding from the U.S. Department
+ * of Energy and the U.S. Government consequently retains certain rights.  As
+ * such, the U.S. Government has been granted for itself and others acting on
+ * its behalf a paid-up, nonexclusive, irrevocable, worldwide license in the
+ * Software to reproduce, distribute copies to the public, prepare derivative
+ * works, and perform publicly and display publicly, and to permit others to do
+ * so.
+ ******************************************************************************/
+
+#include <CL/sycl.hpp>
+#ifndef __HIPSYCL__
+#include <CL/sycl/backend/cuda.hpp>
+#endif
+#include <iostream>
+
+#include "oneapi/mkl/rng/detail/hiplike_helper.hpp"
+#include "rocrand_common.hpp"
+#include "rocrand_helper.hpp"
+#include "oneapi/mkl/exceptions.hpp"
+#include "oneapi/mkl/rng/detail/rocrand/onemkl_rng_rocrand.hpp"
+#include "oneapi/mkl/rng/detail/engine_impl.hpp"
+#include "oneapi/mkl/rng/engines.hpp"
+
+namespace oneapi {
+namespace mkl {
+namespace rng {
+namespace rocrand {
+
+#if !defined(_WIN64)
+class mrg32k3a_impl : public oneapi::mkl::rng::detail::engine_impl {
+public:
+    mrg32k3a_impl(cl::sycl::queue queue, std::uint32_t seed)
```

Comment:
cl::sycl ==> sycl (in all places)

Reply:
changed in all places tests are still passing
[rocrng_ct_tests_still_pass_after_namespace_change.txt](https://github.com/oneapi-src/oneMKL/files/8655409/rocrng_ct_tests_still_pass_after_namespace_change.txt)